mmv1/products/securitycenter/FolderSccBigQueryExports.yaml

```diff
@@ -0,0 +1,138 @@
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Resource
+name: 'FolderSccBigQueryExports'
+base_url: folders/{{folder}}/bigQueryExports
+self_link: folders/{{folder}}/bigQueryExports/{{big_query_export_id}}
+create_url: folders/{{folder}}/bigQueryExports?bigQueryExportId={{big_query_export_id}}
+update_verb: :PATCH
+update_mask: true
+import_format:
+  - 'folders/{{folder}}/bigQueryExports/{{big_query_export_id}}'
+description: |
+  A Cloud Security Command Center (Cloud SCC) Big Query Export Config.
+  It represents exporting Security Command Center data, including assets, findings, and security marks
+  using gcloud scc bqexports
+  ~> **Note:** In order to use Cloud SCC resources, your organization must be enrolled
+  in [SCC Standard/Premium](https://cloud.google.com/security-command-center/docs/quickstart-security-command-center).
+  Without doing so, you may run into errors during resource creation.
+references: !ruby/object:Api::Resource::ReferenceLinks
+  guides:
+    'Official Documentation': 'https://cloud.google.com/security-command-center/docs/how-to-analyze-findings-in-big-query'
+  api: 'https://cloud.google.com/security-command-center/docs/reference/rest/v1/folders.bigQueryExports'
+examples:
+  - !ruby/object:Provider::Terraform::Examples
+    name: 'scc_folder_big_query_export_config_basic'
+    primary_resource_id: 'custom_big_query_export_config'
+    external_providers: ["random", "time"]
+    skip_test: true
+    vars:
+      big_query_export_id: 'my-export'
+      dataset: 'my-dataset'
+      dataset_id: 'my_dataset_id'
+      name: 'my-export'
+      folder_display_name: "folder-name"
+    test_env_vars:
+      org_id: :ORG_ID
+      project: :PROJECT_NAME
+
+parameters:
+  - !ruby/object:Api::Type::String
+    name: folder
+    required: true
+    immutable: true
+    url_param_only: true
+    description: |
+      The folder where Cloud Security Command Center Big Query Export
+      Config lives in.
+  - !ruby/object:Api::Type::String
+    name: bigQueryExportId
+    required: true
+    immutable: true
+    url_param_only: true
+    description: |
+      This must be unique within the organization.
+properties:
+  - !ruby/object:Api::Type::String
+    name: name
+    output: true
+    description: |
+      The resource name of this export, in the format
+      `projects/{{project}}/bigQueryExports/{{big_query_export_id}}`.
+      This field is provided in responses, and is ignored when provided in create requests.
+  - !ruby/object:Api::Type::String
+    name: description
+    description: |
+      The description of the notification config (max of 1024 characters).
```

Comment:
```suggestion
      The description of the export (max of 1024 characters).
```

Reply:
Thank you for the suggestion. Addressed.